pkg/v3/stores.go

```diff
@@ -78,3 +90,59 @@ func WithLimit(limit int) ViewOpt {
 		opts.limit = limit
 	}
 }
+
+type metadataStore struct {
+	ticker       *tickers.BlockTicker
```

Comment:
I think it is fine for now, but I think we shouldn't conflate the store with tickers. Just to maintain consistency, tickers should only be connected to an observer to create observation flows instead.